debian/rules

```diff
@@ -16,6 +16,19 @@ else
     extra_substvars = -VbionicOrLater:Depends=""
 endif
 
+# and then do the same thing for awscli - of which we used to have a patched version called aws-cli pre-jammy.
+# once jammy boxes are the oldest things we install clusterman on, we can get rid of this and just include this
+# directly in debian/control
+ifeq ($(shell (. /etc/os-release && dpkg --compare-versions $$VERSION_ID "ge" "22.04" && echo yes || echo no)),yes)
+    extra_substvars = -VjammyOrLater:Depends="awscli"
+else
+	# aws-cli only exists internally, so lets make sure that we only use it internally
+    ifneq ($(shell echo ${PAASTA_ENV}), YELP)
+        extra_substvars = -VjammyOrLater:Depends="awscli"
+    endif
+    extra_substvars = -VjammyOrLater:Depends="aws-cli"
```

Comment:
Does this mean that `aws-cli` is used regardless and `awscli` is only used when outside Yelp? I guess I'm a bit confused why it's written this way around instead of swapping the two (and using an `ifeq` instead of `ifneq`), since it looks like this way around that `aws-cli` is always used, even outside Yelp.

Reply:
err, i dun' goofed here - meant to add an else

Reply:
(i'll also swap the conditional)

Reply:
Also do we actually *need* our forked version of awscli?

Reply:
we don't - i mostly just don't want to deal with updating the puppet on all our pre-jammy boxes to use a non-forked version (since that's a ton of boxes)

Reply:
oh right, this is for the clusterman-cli not for the part that runs in a container